<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBaseMapConversionLevels</class>
 <widget class="QWidget" name="FormBaseMapConversionLevels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>214</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSliderBaseMapWeight">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBaseToOthersProcessing">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_23">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSliderBMNoIters">
          <property name="toolTip">
           <string>Set the number of iterations of the procedure.</string>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Amplitude</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Iters</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Edges </string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="horizontalSliderConversionBaseMapMixing">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the blending with the non processed normal image (ie. base image after sobel operator applied). This parameter changes the weight of the blending on edges. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="horizontalSliderBaseMapAmplidute">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the amplitude of the normals. &lt;/p&gt;&lt;p&gt;Sometimes very small value of the amplitude parameter helps to obtain better image. Don't worry if the image you see is totally blue. You can change the &lt;span style=&quot; font-weight:600;&quot;&gt;normal step&lt;/span&gt; parameter after conversion in the &lt;span style=&quot; font-weight:600;&quot;&gt;normal&lt;/span&gt; tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="minimum">
           <number>-300</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Flatting</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSliderConversionBMFilterRadius">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the normal extraction radius. Large radius = larger extraction but also larger blur of image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="singleStep">
           <number>0</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Pre smooth</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="horizontalSliderConversionBaseMapFlatness">
          <property name="toolTip">
           <string>It makes &quot;hills&quot; more flat.</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSliderConversionBMPreSmoothRadius">
          <property name="toolTip">
           <string>Smooth the image before processing.</string>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_33">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>Blending</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="horizontalSliderConversionBaseMapBlending">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the blending with the non processed normal image (ie. base image after sobel operator applied) but here standart mixing is applied.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderConversionBMPreSmoothRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_33</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConversionBMFilterRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_29</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBMNoIters</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_27</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBaseMapAmplidute</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_23</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConversionBaseMapFlatness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_25</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConversionBaseMapMixing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_16</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConversionBaseMapBlending</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_31</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBaseMapWeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
